### PR TITLE
doc: Add chapter on linking

### PR DESCRIPTION
**Pull request by matthiasbeyer**
Chapter on how linking to {internal, external} content works.

- [x] Linking overall
- [x] Linking internal content
- [X] Linking external content

### CODE REVIEW

**Comment by matthiasbeyer** [comment]
@neithernut , @TheNeikos Comments are welcome! How shall we define this? PRs are welcome as well!

**Comment by matthiasbeyer** [comment]
@TheNeikos We discussed the idea of weak and strong linking, right? I cannot remember what the concensus was... can we have a discussion about this, please? And maybe get some definitions together so we can finalize this chapter of the document?

**Comment by matthiasbeyer** [comment]
waiting on #194

File: doc/src/02010-linking.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+## Linking from an store entry {#sec:thestore:linking}
+
+In @sec:thestore:fileformat:header:imag was already defined that there MUST BE a
```
Comment (neithernut) [inline doc/src/02010-linking.md]: This line lacks some word. E.g. `... it was defined that`.

File: doc/src/02010-linking.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+section "imag" in the header. This section can be used to link to "internal" and
+"external" content, whereas "internal content" refers to entries which are
+stored in the very same store as the entry which links.
+The term "external content" means content which is not stored in the in the
```
Comment (neithernut) [inline doc/src/02010-linking.md]: Double `in the`

File: doc/src/02010-linking.md (hunk 1 above)
Comment (matthiasbeyer) [inline doc/src/02010-linking.md]: What is wrong with `"In @sec... was already defined that there MUST BE a section "imag" in the header"` ?

Comment (neithernut) [inline doc/src/02010-linking.md]: Well... the sentence isn't complete.

**Comment by matthiasbeyer** [comment]
As nobody seems to care anymore about strong and weak linking, I will remove these parts from the doc and merge this branch afterwards.

**Pull request merged**